packages/beacon-node/test/e2e/eth1/jsonRpcHttpClient.test.ts

```diff
@@ -198,8 +197,9 @@ describe("eth1 / jsonRpcHttpClient - with retries", () => {
         },
       })
     ).rejects.toThrow("getaddrinfo ENOTFOUND");
-    expect(retryCount).toBeWithMessage(retries, "ENOTFOUND should be retried before failing");
-  });
+    expect(retryCount).toBe(retries);
+    expect("code ECONNREFUSED should be retried before failing").toBeTruthy();
```

Comment:
Non-empty string will always be truthy. This line doesn't do anything, as well as other similar lines in this file

Reply:
Alright, noted!